bridgerelayer/chain/substrate/connection.go

```diff
@@ -0,0 +1,65 @@
+package substrate
+
+import (
+	"context"
+
+	"github.com/sirupsen/logrus"
+
+	gsrpc "github.com/snowfork/go-substrate-rpc-client"
+	"github.com/snowfork/go-substrate-rpc-client/signature"
+	"github.com/snowfork/go-substrate-rpc-client/types"
+)
+
+// Connection ...
```

Comment:
ditto on method descriptions, the `...` was a placeholder meant to updated on the next significant refactor (i.e. this PR)